src/data/nat/log.lean

```diff
@@ -64,4 +93,21 @@ end
 lemma pow_log_le_self (b x : ℕ) (hb : 1 < b) (hx : 1 ≤ x) : b ^ log b x ≤ x :=
 by rw [pow_le_iff_le_log _ _ hb hx]
 
+lemma log_le_log_of_le {b n m : ℕ} (h : n ≤ m) : log b n ≤ log b m :=
+begin
+  cases le_or_lt b 1 with hb hb,
+  { rw log_eq_zero_of_le hb; exact zero_le' },
+  cases eq_zero_or_pos n with hn hn,
```

Comment:
I'd be inclined to begin new braces here to handle the second case. Even though there is only one goal remaining, the use of braces makes it easier to understand that you're still dealing with cases.

Reply:
I was actually pondering what to do in that case (pun not intended), thanks for clarifying!